docs/about-filecoin/ipfs-and-filecoin.md

```diff
@@ -5,31 +5,31 @@ description: Learn more about the relationship and different use-cases between I
 
-The lack of a built-in incentive mechanism is the challenge that Filecoin hopes to solve by allowing users to incentivize long-term distributed storage at competitive prices through a marketplace of storage nodes, while maintaining the efficiency and resiliency provided by the IPFS network.
+The lack of a built-in incentive mechanism is the challenge Filecoin hopes to solve by allowing users to incentivize long-term distributed storage at competitive prices through a marketplace of storage nodes while maintaining the efficiency and resiliency provided by the IPFS network.
 
 ## Different persistence strategies: which should I use?
 
 Depending on your use case, one or multiple of these persistence strategies might be a good fit for you.
 
 ### Using IPFS
 
-As mentioned: in IPFS, data is hosted by nodes pinning the desired data. To persist data when a user’s node is offline - users must either rely on other peers to voluntarily/altruistically pin their data, or use a centralized pinning service (often running long lived host nodes on a cloud-service provider) to store the data.
+As mentioned: in IPFS, data is hosted by nodes pinning the desired data. To persist data when a user's node is offline, users must either rely on other peers to voluntarily/altruistically pin their data or use a centralized pinning service to store it.
 
 Relying on peers to altruistically cache data might work well where one -- or multiple -- organizations share popular files on an internal network (intranet), or where strong social contracts can be used to ensure the content remains hosted and maintained long-term. Most users in the IPFS Network use a pinning service.
 
 ### Using Filecoin
 
-The final option is to pin your data to a decentralized storage marketplace, such as Filecoin. In Filecoin's structure, clients make regular small payments to store data at a specified availability, while miners earn these payments by continuously verifying this data's integrity, storing it, and ensuring it can be retrieved quickly. This allows users to incentivize Filecoin miners to ensure their content will be live when it's needed, a clear advantage over depending solely on the generosity of other network users as required with using IPFS on its own.
+The final option is to pin your data to a decentralized storage marketplace, such as Filecoin. In Filecoin's structure, clients make regular small payments to store data at a specified availability. Simultaneously, miners earn these payments by continuously verifying this data's integrity, storing it, and ensuring it can be retrieved quickly. This allows users to incentivize Filecoin miners to ensure their content will be live when it's needed, a clear advantage over depending solely on the generosity of other network users as required with using IPFS on its own.
```

Comment:
Not related to the PR, but note:

It's highly questionable to claim that PoSt "ensures it can be retrieved quickly". It ensures it exists, in some replicated form, on the miner's storage. Fast retrieval is, uh, not promised.

Reply:
Yeah, I'm going to run a content review of this. Below it claims that Filecoin is built "on top of IPFS". And while I can accept there might be a way to see it like that, this leads to huge amounts of confusion among people learning about things.

Reply:
https://github.com/filecoin-project/filecoin-docs/pull/477